python/cugraph/cugraph/tests/layout/test_force_atlas2.py

```diff
@@ -138,74 +190,3 @@ def test_force_atlas2(graph_file, score, max_iter, barnes_hut_optimize):
     assert test_callback.on_epoch_end_called_count == max_iter
     # verify `on_train_end` was only called once
     assert test_callback.on_train_end_called_count == 1
-
-
-# FIXME: this test occasionally fails - skipping to prevent CI failures but
-# need to revisit ASAP
```

Comment:
I'm fine removing a flaky test, but should there at least be a FIXME documenting the missing coverage?  Better yet, and issue?

Reply:
issue created and fixme added.
https://github.com/rapidsai/cugraph/issues/3900